.github/workflows/ci.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --all-features
+          args: --features ftdi,build-binary
```

Comment:
This is because tests fail if the `skip_serializing_if` on `probe-rs-targets` gets enabled.